docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
 # View
 
-The `View` class is the base class for all views. With this class you can manage the background color and images for UI components.
+[View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) is the fundamental concept for the all components such as `Button`, `Image`, `Text`, and so on.
+`View` provides [Properties](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#properties), [Methods](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#methods), and [Events](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#events) that are commonly used on every components. To show the `View`, you must add `View` on the [Window](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Window.html) as a child.
```

Comment:
Please remove "the" before "View".
Please consider the same in line number 13.

Reply:
Thank you I fixed it.
But if some cases that the View is used as a specified View object, I remained a or the.

Reply:
Thank you for updating the comment.

It is recommended to avoid articles before a noun.
If it is "View object" then we can use "the" or "a".
For example, "If a `View` object is... ", "The `ParentOrigin` property of `View`...."
Here, we are considering the object and the property. 
Another example is, Tizen is a noun. Therefore, we won't use articles with Tizen. But when it is a Tizen application, we can use the article.
"The Tizen application" or "a Tizen application"
Here the application is considered, so usage of the article is fine. In other cases, let's not use articles.

I hope this clarifies the doubt. Please let me know if you need more details.

Reply:
Thank you. I fixed this expression slightly.

line 13
For example, if a `View` object has properties `Position` (5, 3) and `ParentOrigin` is *Center*, then the object is located (5, 3) away from *Center* position of its parent `View` object.

line 43
In the final calculation of `View`, `Orientation` and `Scale` of parents are first applied to child `View` with parent's `PivotPoint`, and then `Orientation` and `Scale` of child are applied.